### ISSUE TITLE
Brain life support 

### ISSUE THREAD

**Issue by quandaledongle** [studio-minus/ppg-bugs-and-requests#2813]
### Feature description

We already have a life support for the heart but i want also for the brain so a braindead person can still live😅. Maybe do this with some kind of super computer model? Linked to the head with a blood cable

### Feature purpose

It is a fun new mechanic that might even be used to heal dead people😅 depends how this feature will be build (if so ofcourse)

**Comment by username90001** [comment]
> 

brain dead person cant be alive even if body of brain dead person is alive the person is dead and near impossible to revive and also brain dead human cant have consciousness so body will be ragdoll + he wont be able to breath + hearth dont work 
brain dead human is very dead human

**Comment by quandaledongle** [comment]
> > 
> 
> 
> 
> brain dead person cant be alive even if body of brain dead person is alive the person is dead and near impossible to revive and also brain dead human cant have consciousness so body will be ragdoll + he wont be able to breath + hearth dont work 
> 
> brain dead human is very dead human

I want to be able to fix the brain somehow on a machine so no more braindead you knwo?

**Comment by username90001** [comment]
> > > 
> > 
> > 
> > brain dead person cant be alive even if body of brain dead person is alive the person is dead and near impossible to revive and also brain dead human cant have consciousness so body will be ragdoll + he wont be able to breath + hearth dont work
> > brain dead human is very dead human
> 
> I want to be able to fix the brain somehow on a machine so no more braindead you knwo?

actualy trying make human not brain dead means collect all of his parts of brain to make output same basicaly output of brain must be same as it was or something like this so if this true then it means that if human die he get revived in very far future due to unknown reasons like tech or life appear on other planet far away but life is same as earth one
actualy u can fix brain dead in game using tritium or electricity that can burn humans too

**Comment by thumbthumb9** [comment]
> > > 
> > 
> > 
> > brain dead person cant be alive even if body of brain dead person is alive the person is dead and near impossible to revive and also brain dead human cant have consciousness so body will be ragdoll + he wont be able to breath + hearth dont work
> > brain dead human is very dead human
> 
> I want to be able to fix the brain somehow on a machine so no more braindead you knwo?

You dont need a machine, Inject a little tritium (which you can get from the nukes) into the head and the patient's brain will be fixed

**Comment by username90001** [comment]
> > > > 
> > > 
> > > 
> > > brain dead person cant be alive even if body of brain dead person is alive the person is dead and near impossible to revive and also brain dead human cant have consciousness so body will be ragdoll + he wont be able to breath + hearth dont work
> > > brain dead human is very dead human
> > 
> > 
> > I want to be able to fix the brain somehow on a machine so no more braindead you knwo?
> 
> You dont need a machine, Inject a little tritium (which you can get from the nukes) into the head and the patient's brain will be fixed

yeah i said about tritium but problem tritium can hurt humans alot so electricity is nearly better

**Comment by B-Knots** [comment]
Honestly i just want the head to have life signs as it shouldnt rot when blood is pumped to it

**Comment by quandaledongle** [comment]
No> > > > 
> 
> > > 
> 
> > > 
> 
> > > brain dead person cant be alive even if body of brain dead person is alive the person is dead and near impossible to revive and also brain dead human cant have consciousness so body will be ragdoll + he wont be able to breath + hearth dont work
> 
> > > brain dead human is very dead human
> 
> > 
> 
> > I want to be able to fix the brain somehow on a machine so no more braindead you knwo?
> 
> 
> 
> You dont need a machine, Inject a little tritium (which you can get from the nukes) into the head and the patient's brain will be fixed

Pretty sure that doesnt work i just tried it

**Comment by username90001** [comment]
> Honestly i just want the head to have life signs as it shouldnt rot when blood is pumped to it

if u want make dead body move then u need just rotors servos and other 
dead brain = absolutly dead human
